samples/notebooks/python/08-native-function-inline.ipynb

```diff
@@ -38,33 +41,75 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 15,
    "id": "1da651d4",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: semantic-kernel==0.2.7.dev0 in c:\\users\\mattm\\appdata\\local\\programs\\python\\python310\\lib\\site-packages (0.2.7.dev0)\n",
```

Comment:
It doesn't look like something that we should check in, please correct me if I'm wrong :)

Reply:
Yes, my mistake. This change was from another branch I was working on. I have reverted changes to the notebook.
This is supposed to be very simple change of single string 😅